indico/modules/rb/operations/bookings.py

```diff
@@ -191,16 +202,16 @@ def get_room_calendar(start_date, end_date, room_ids, include_inactive=False, **
     end_dt = datetime.combine(end_date, time(hour=23, minute=59))
     query = _bookings_query(dict(filters, start_dt=start_dt, end_dt=end_dt, room_ids=room_ids,
                                  include_inactive=include_inactive))
-    query = query.order_by(db.func.indico.natsort(Room.full_name))
+    bookings = query.order_by(db.func.indico.natsort(Room.full_name)).all()
     rooms = (Room.query
-             .filter(~Room.is_deleted, Room.id.in_(room_ids) if room_ids else True)
+             .filter(~Room.is_deleted, Room.id.in_(room_ids))
```

Comment:
why did you remove it? if `room_ids` is None this will now fail (easy to reproduce by just sending a GET request to the calendar endpoint so no room ids are included)